<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>994</width>
    <height>504</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>ShearDrivenMaster</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,0,9">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_mainTitle"/>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::HLine</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="9,1">
      <item>
       <layout class="QVBoxLayout" name="LayoutMainView" stretch="4,5,1">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="LayoutDevices" stretch="1,1,1">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QWidget" name="WidgetImager" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">#WidgetImager{border: 1px solid rgb(0, 170, 255)}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetMinimumSize</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="LabelImager">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(135, 206, 235);</string>
               </property>
               <property name="text">
                <string>光机</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="LayoutImager"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="WidgetRobot" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">#WidgetRobot{border: 1px solid rgb(0, 170, 255)}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="LabelRobot">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(135, 206, 235);</string>
               </property>
               <property name="text">
                <string>机械臂</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="LayoutRobot"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="WidgetCoater" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">#WidgetCoater{border: 1px solid rgb(0, 170, 255)}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="LabelCoater">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(135, 206, 235);</string>
               </property>
               <property name="text">
                <string>涂布机</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="LayoutCoater"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="LayoutInfo" stretch="3,2">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QWidget" name="WidgetSequencing" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">#WidgetSequencing{border: 1px solid rgb(0, 170, 255)}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="LabelSequencing">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(135, 206, 235);</string>
               </property>
               <property name="text">
                <string>测序信息</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="LayoutSequencing"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="WidgetMessage" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">#WidgetMessage{border: 1px solid rgb(0, 170, 255)}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,9">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="LabelMessage">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(135, 206, 235);</string>
               </property>
               <property name="text">
                <string>运行信息</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="LayoutMessage"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="LayoutRunButtons" stretch="2,2,2,0,0">
          <property name="spacing">
           <number>15</number>
          </property>
          <item>
           <widget class="QPushButton" name="ButtonStartRun">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color: rgb(0, 85, 0);
	background-color: rgb(178,224,230);
	border-radius:9px;
}

QPushButton:hover
{
	background-color: rgb(135,206,235);
}

QPushButton:pressed
{
	background-color: rgb(30,144,255);
	color: white;
}

QPushButton:disabled
{
	background-color: rgb(192,192,192);
}</string>
            </property>
            <property name="text">
             <string>开始运行</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonPauseRun">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
	color: rgb(0, 0, 255);
	background-color: rgb(178,224,230);
	border-radius:9px;
}

QPushButton:hover
{
	background-color: rgb(135,206,235);
}

QPushButton:pressed
{
	background-color: rgb(30,144,255);
	color: white;
}

QPushButton:disabled
{
	background-color: rgb(192,192,192);
}</string>
            </property>
            <property name="text">
             <string>暂停运行</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ButtonStopRun">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>400</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">
QPushButton
{
	color: rgb(255, 0, 0);
	background-color: rgb(178,224,230);
	border-radius:9px;
}

QPushButton:hover
{
	background-color: rgb(135,206,235);
}

QPushButton:pressed
{
	background-color: rgb(30,144,255);
	color: white;
}

QPushButton:disabled
{
	background-color: rgb(192,192,192);
}</string>
            </property>
            <property name="text">
             <string>紧急停止</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
            <item>
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>回收Cycle:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="LineSycleNum">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="ButtonSetCycle">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>40</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{
	background-color: rgb(178,224,230);
	border-radius:9px;
}

QPushButton:hover
{
	background-color: rgb(135,206,235);
}

QPushButton:pressed
{
	background-color: rgb(30,144,255);
	color: white;
}

QPushButton:disabled
{
	background-color: rgb(192,192,192);
}</string>
              </property>
              <property name="text">
               <string>设置</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="LayoutOthetButtons" stretch="2,2,2,2,1,2,2,2,2">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QPushButton" name="ButtonChipSeting">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>本地数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonImagerSeting">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>光机设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonRobotSeting">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>机械臂设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonCoaterSeting">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>涂布机设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonSystemInit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>系统初始化</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonAlarmReset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>报警重置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonTakePhotos">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>一键拍照</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonQueryLogs">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>错误信息</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
